convert2rhel/pkgmanager/handlers/dnf/__init__.py

```diff
@@ -86,6 +86,11 @@ def _set_up_base(self):
         # going in with the packages in the transaction.
         self._base._ds_callback = DependencySolverProgressIndicatorCallback()
 
+        # Override the exclude option that is loaded from the config and set it to empty.
+        # NOTE(r0x0d): In DNF they have exclude to be `py:excludepkgs`. It is fine to use `exclude` for now as it is
```

Comment:
Small nitpick but some tools that consolidate NOTE/TODO etc won't find it like this, so I've started doing this instead when I can:

```suggestion
        # NOTE @r0x0d: In DNF they have exclude to be `py:excludepkgs`. It is fine to use `exclude` for now as it is
```